src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operator) => {
+  const regex = new RegExp('[.]{2,}');
+
+  if (regex.test(numberOne) || regex.test(numberTwo)) {
+    return 'error';
+  }
+  const x = numberOne === null ? Big(0) : Big(numberOne);
+  const y = numberTwo === null ? Big(0) : Big(numberTwo);
+
+  switch (operator) {
+    case '+':
+      return x.plus(y).toString();
+
+    case '-':
+      return x.minus(y).toString();
+
+    case 'x':
+      return x.times(y).toString();
+
+    case '/':
+      return x.div(y).toString();
+
```

Comment:
Hey, what will happen if the second number is `0` ? i think it will blow up, let's handle it.